src/args/lint-python/builder.sh

```diff
@@ -20,7 +20,8 @@ function main {
     && tmpdir="$(mktemp -d)" \
     && copy "${package_path}" "${tmpdir}/${package_name}" \
     && pushd "${tmpdir}" \
-    && mypy --config-file "${envSettingsMypy}" "${package_name}" \
+    && dmypy start -- --config-file "${envSettingsMypy}" \
```

Comment:
Don't we have to turn it off afterwards?

Reply:
@dsalaza4  No, it's better to turn it off once all the dirs were checked, to maximize the re-utilization of previous runs, https://mypy.readthedocs.io/en/stable/mypy_daemon.html#mypy-daemon-mypy-server

Reply:
Also, I was getting some errors about the config file if the daemon was turned off after the `package_name` check